content/en/user-manual/xr/ar/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: AR
+layout: usermanual-page.hbs
+position: 2
+---
+
+![AR View][1]
```

Comment:
The replacement image is 1.7MB PNG. Can you reduce the size and turn it into a JPG please if we are keeping that. I think you mentioned using a GIF too?

Reply:
Yes, I was going to try replacing this image with a GIF from my device. If I manage to record smooth footage of the PlayCanvas cube to post as the GIF I'll replace it with that. Otherwise I'll go ahead and just minify the existing image.